### ISSUE TITLE
Run time error after update to MudBlazor Version 5.1.5

### ISSUE THREAD

**Issue by T-Alkathiri** [fullstackhero/blazor-starter-kit#343]
Dear Mukesh
Thank you very much for the greate template for BlazorHero.
I create a new project with the template and every thing works fine (with MudBlazor version 5.1.4).
But when Updated MudBlazor to version 5.1.5 I got the following run time error :
System.MissingMethodException: Method not found: void MudBlazor.Palette.set_Primary(string)

crit: Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: The type initializer for 'BlazorHero.CleanArchitecture.Client.Infrastructure.Settings.BlazorHeroTheme' threw an exception.

Could you please kindly help me solving this issue, because I really liked the template and want to continue for all upcoming projects with it.
But this error confused me and can not make any decision weather to continue with Blazor Hero or just skip it.
Thank you very much for your time and effort.

**Comment by yadoy666** [comment]
Hi @T-Alkathiri , have you try to change <MudToolBarSpacer> and <MudAppSpacer> to <MudSpacer> ? In my case i have successfully update   mudblazor with the latest version with a little change above. And don't forget to force reload your browser (CTRL + F5) after making changes.

**Comment by T-Alkathiri** [comment]
Yes I have changed the ? to the latest version, and I did (CTRL + F5) in the browser but I still have the error.
BlazorHero.CleanArchitecture.Client.Infrastructure.Settings.BlazorHeroTheme' threw an exception.
Thank you very much.
